build/rust/Cargo.lock

```diff
@@ -677,9 +704,9 @@ dependencies = [
 
 [[package]]
 name = "digest"
-version = "0.10.3"
+version = "0.10.6"
```

Comment:
is this version bump necessary? Please don't make any Cargo.lock changes that aren't absolutely necessary